docs/crio.8.md

```diff
@@ -261,6 +262,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
     Kubernetes configuration are considered. Bind mounts that CRI-O
     inserts by default (e.g. '/dev/shm') are not considered. (default: "/usr/share/containers/oci/hooks.d")
 
+**--hostnetwork-disable-selinux**: Determines whether SELinux should be disabled within a pod when it is running in the host network namespace.
```

Comment:
It would probably be good to at least add a warning that turning SELinux off is not a recommended configuration.

Reply:
Remind me why running in the host network would ever require you to disable SELinux separation?  It shouldn't require that, as far as I can tell.

Reply:
This was introduced in this commit

https://github.com/cri-o/cri-o/commit/349b85a7f2b97d23fd65490289bf5d66e888e0f9

In this PR, we are introducing a new flag to enable/disable the selinux for pod by not breaking what's already there.